kms/api/CloudKmsAspNetSample/CloudKmsAspNetSample.csproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Google.Cloud.Kms.V1" Version="1.0.0-beta03" />
+    <PackageReference Include="Microsoft.AspNetCore.App" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Update="appsecrets.json.encrypted" CopyToOutputDirectory="PreserveNewest" />
+    <None Update="appsecrets.json.keyname" CopyToOutputDirectory="PreserveNewest" />
```

Comment:
Are you storing them as two separate files because of gcloud command limitation?
Why not having a single file that has 1 entry in clear text (name of the key) and another entry is encryption of the configuration?

Reply:
Yes, I store them in two separate files to make it easy to use with the gcloud command.  A combined file would be better, but more difficult to use from gcloud.